tags

```diff
@@ -0,0 +1,10 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```

Comment:
I see! Got it. So I figured it should have been in `.gitignore` then. (I see you already realised it in https://github.com/mikaelbr/open-source-ideas/pull/24#discussion_r85046228)